web-wallet/src/lib/dusk/number/getDecimalSeparator.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @returns {string}
+ */
+
+const getDecimalSeparator = () => {
```

Comment:
Wouldn't a `(.1).toLocaleString().slice(1,2)` one-liner be simpler?

Reply:
Simpler, yes, but I think that using the Intl API might be the correct way.